.evergreen/config.yml

```diff
@@ -1399,7 +1399,7 @@ buildvariants:
   display_name: "${sanitize} ${os-fully-featured} ${ssl-auth}"
   matrix_spec:
     os-fully-featured: "ubuntu-18.04"
-    swift-version: "5.5"
+    swift-version: "5.4" # TODO SWIFT-1499: update to Swift 5.5
```

Comment:
filed SWIFT-1499 to track eventually updating this to 5.5 once relevant Swift issues are fixed. we could have stayed with 5.5 and updated the test scripts so that we would run every suite besides the async/await ones, but that would have required either a very long/ugly regex to match every suite name that does not contain "AsyncAwait", or pulling out the async/await tests into their own target, so this seemed like a reasonable solution for now